applications/ecallmgr/src/ecallmgr.hrl

```diff
@@ -256,9 +256,9 @@
                               ,{<<"Media-Encryption-Enforce-Security">>,<<"sdp_secure_savp_only">>}
                               ,{<<"Media-Files-Separator">>, <<"playback_delimiter">>}
                               ,{<<"Media-Webrtc">>, <<"media_webrtc">>}
-                              ,{<<"Origination-Call-ID">>, <<"sip_origination_call_id">>}
+                              ,{<<"Origination-Call-ID">>, <<"origination_uuid">>}
```

Comment:
why this change ?

Reply:
@lazedo I can't find any mention of `sip_origination_call_id` in FreeSWITCH wiki nor did I see the asked-for call-id being set, but setting `origination_uuid` worked and is found in [Channel Variables](https://freeswitch.org/confluence/display/FREESWITCH/Channel+Variables)

Reply:
@jamesaimonetti its set by us so that we can bind to a know call-id from the originating process (fax worker uses it) for calls that have 2+ destinations (2+ /= uuids) and the first one fails we can still get events from the other legs.
ecallmgr_call_event sets `{<<"Origination-Call-ID">>, kzd_freeswitch:origination_call_id(Props)}` from that value and `kapi_call:publish_event` prefers it when setting the routing key.
i propose we leave `Origination-Call-ID` as-is and change fax_worker to send that instead of `Outbound-Call-ID` and have stepswitch forward `Origination-Call-ID` and  `Outbound-Call-ID` (with value = `Origination-Call-ID).

this can also be useful to track quickcall